Sources/MarathonCore/PackageManager.swift

```diff
@@ -394,7 +394,7 @@ public final class PackageManager {
         if toolsVersion >= Version(major: 4, minor: 2) {
             var versionString = String(toolsVersion.major)
 
-            if toolsVersion.minor > 0 {
+            if toolsVersion == Version(major: 4, minor: 2) {
```

Comment:
This is the only known case where the `".\(toolsVersion.minor)"` is needed, and given this project is going to be deprecated, there could not be the need to handle any other case, but this fixes

```
<unknown>:0: error: invalid value '5.1' in '-swift-version 5.1'
<unknown>:0: note: valid arguments to '-swift-version' are '4', '4.2', '5'
```